src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./ColorButton.scss";
+
+const ColorButton = (props) => {
+  return (
+    <div>
+     <button 
+        className={`colorButton ${props.color}`} 
+        onClick={() => props.setColor(props.color)}
+      ></button>
+    </div>
+  );
```

Comment:
```suggestion
const ColorButton = ({color, setColor}) => {
  return (
    <div>
     <button 
        className={`colorButton ${color}`} 
        onClick={() => setColor(color)}
      ></button>
    </div>
  );
```

wrap up 시간에 배웠던 객체 구조분해 할당 적용해주세요!